### PR TITLE
PT-287/Add transitions to dashboards

### PR DESCRIPTION
**Pull request by florianmski**
> Tracked in JIRA by [PT-287](https://novoda.atlassian.net/browse/PT-287)

### Scope of the PR

Add transitions when changing dashboards, a simple fade-in/out for now.

### CODE REVIEW

**Comment by florianmski** [comment]
❌ 

As you can see below the slack cards are now not taking the whole width, I already spent some time trying to fix the CSS but I don't know much :(
If someone has an idea...

webdashboard

**Comment by frapontillo** [comment]
We can pair on this tomorrow afternoon (I'm off in the morning) if you still haven't figured it out.

**Comment by florianmski** [comment]
Removing ❌ CSS fixed 😄

**Comment by ouchadam** [comment]
no gif? 🐤 

I trust you though!

File: angular2-client/src/app/dashboard-carousel/dynamic.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.dashboard {
+  display: block;
```
Comment (frapontillo) [inline angular2-client/src/app/dashboard-carousel/dynamic.component.scss]: Yeeeeeeah. That's usually the culprit 😆

**Comment by frapontillo** [comment]
Good work, maybe a good idea would be to migrate to flexbox instead of using percentage `width` and `height` **IN A REMOTE FUTURE**.

**Comment by florianmski** [comment]
@frapontillo 💯 on some flexbox and would love to pair on that, completely lost with CSS 😄

**Pull request merged**